lib/api/controllers/documentController.js

```diff
@@ -408,10 +478,11 @@ class DocumentController extends BaseController {
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  validate(request) {
+  validate (request) {
     assertHasBody(request);
     assertHasIndexAndCollection(request);
 
+    // @todo validation service should not take request argument
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)